auth-server/src/utils/getPostCommentLink.ts

```diff
@@ -0,0 +1,15 @@
+// Copyright 2019-2020 @paritytech/polkassembly authors & contributors
+// This software may be modified and distributed under the terms
+// of the Apache-2.0 license. See the LICENSE file for details.
+
+import { CommentCreationHookDataType, PostType } from '../types';
+
+/**
+ * Get Post link type from an event
+ */
+export default (type: PostType, data: CommentCreationHookDataType): string => {
+	const DOMAIN = process.env.DOMAIN_NAME && process.env.DOMAIN_PROTOCOL ? `${process.env.DOMAIN_PROTOCOL}${process.env.DOMAIN_NAME}` : 'https://test.polkassembly.io';
```

Comment:
I would log an error here, because it will not help much to have a default to test.

Reply:
done